lib/use-cases/SendConfirmationEmail.js

```diff
@@ -0,0 +1,16 @@
+module.exports = ({ gateways: { dropbox: dropboxes, notify } }) => {
+  return async dropboxId => {
+    try {
+      const dropbox = await dropboxes.get(dropboxId);
+      if (!dropbox) throw new Error('Invalid dropbox');
+      await notify.send({
+        email: dropbox.parentsEmail,
+        referenceId: dropboxId,
+        templateId: process.env.GOV_NOTIFY_CONFIRMATION_TEMPLATE_ID,
+        templateVars: { feedbackFormUrl: process.env.FEEDBACK_FORM_URL }
```

Comment:
Should we add Childs name into personalisation and state their name in the email? (might be useful if submitting docs for several children)
Also would it be useful to send the link to the dropbox as well in case they remember to add more docs?

Reply:
The template we've been given doesn't have any of that

Reply:
Ah I see